src/config.rs

```diff
@@ -515,7 +515,7 @@ impl Default for DbConfig {
             wal_size_limit: ReadableSize::kb(0),
             max_total_wal_size: ReadableSize::gb(4),
             max_background_jobs: 6,
-            max_manifest_file_size: ReadableSize::mb(20),
+            max_manifest_file_size: ReadableSize::mb(128),
```

Comment:
do we need to adjust Raft DB?

Reply:
It doesn't matter, RaftDB has only a few files.